feelpp/pyfeelpp/feelpp/operators.cpp

```diff
@@ -72,6 +73,16 @@ void defOperator( py::module& m )
             return a.matrixPtr(); },
         py::arg( "trial" ), py::arg( "test" ), py::arg( "range" ), py::arg( "coeff" )=std::string{"1"}, "create a mass matrix" );
     m.def(
+        "mass", []( space_ptr_t const& domain, space_ptr_t const& image, faces_iterator_range_t const& r, std::string const& c = "1" )
```

Comment:
the code is redundant with element wise mass lambda function.
this needs to be made generic wrt elements and faces ranges

Reply:
this is from pyrb #1787